platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init

```diff
@@ -39,35 +39,31 @@ start)
         modprobe leds-dx010
         modprobe lm75
 
-        found=0
-        for devnum in 0 1; do
-                devname=`cat /sys/bus/i2c/devices/i2c-${devnum}/name`
-                # iSMT adapter can be at either dffd0000 or dfff0000
-                if [[ $devname == 'SMBus iSMT adapter at '* ]]; then
-                        found=1
-                break
-                fi
-        done
+        # Detect iSMT SMBus
+        ismt_bus=$(i2cdetect -l|grep 'SMBus iSMT'| cut -c5)
```

Comment:
This will only retrieve once character. Is there a chance that the bus number could potentially end up with two digits (e.g., `i2c-10`)?

Reply:
@jleveque , No, It will only be 0 and 1.